test/functional/mempool_limit.py

```diff
@@ -92,8 +92,8 @@ def test_mid_package_eviction_success(self):
         assert_equal(node.getrawmempool(), [])
 
         # Restarting the node resets mempool minimum feerate
-        assert_equal(node.getmempoolinfo()['minrelaytxfee'], Decimal('0.00001000'))
-        assert_equal(node.getmempoolinfo()['mempoolminfee'], Decimal('0.00001000'))
+        assert_equal(node.getmempoolinfo()['minrelaytxfee'], Decimal('0.00000100'))
```

Comment:
If you need to update, perhaps you could also use the approach with

`FEERATE_100SAT_KVB = Decimal("0.00000100")`

in `test/functional/mempool_limit.py`.